client/src/components/feedback/edit-question-for-queue-modal.tsx

```diff
@@ -0,0 +1,278 @@
+                          }}
+                        >
+                          <option selected disabled>
+                            --Select a category--
+                          </option>
+                          {selectedSubject.categories.map((category) => (
+                            <option
+                              data-cy={`Category-option-${category.id}`}
+                              key={category.name}
+                              value={category.id}
+                            >
+                              {category.name}
+                            </option>
+                          ))}
+                        </Select>
+                      </FormControl>
+                    ) : undefined}
+                  </div>
+
+                  {selectedSubject ? (
+                    <div
+                      className={classes.inputField}
+                      style={{ textAlign: "left" }}
+                    >
+                      <Autocomplete
+                        // an autocomplete kwd-style area for "topics"
+                        data-cy="topic-selector"
+                        multiple
+                        options={selectedSubject?.topics || []}
+                        getOptionLabel={(option) => option.name}
+                        onChange={(e, v) => {
+                          setSelectedTopics(v);
+                        }}
+                        style={{ minWidth: 300 }}
+                        renderOption={(option) => (
+                          <Typography
+                            align="left"
+                            data-cy={`Topic-option-${option.id}`}
+                          >
+                            {option.name}
+                          </Typography>
+                        )}
+                        renderInput={(params) => (
+                          <TextField
+                            {...params}
+                            variant="outlined"
+                            label="Select topic(s)"
+                          />
+                        )}
+                      ></Autocomplete>
+                    </div>
+                  ) : undefined}
+                </Grid>
+                <Button
+                  onClick={handleClose}
+                  data-cy="modal-close-btn"
+                  variant="contained"
+                  color="primary"
+                  component="span"
+                >
+                  Cancel
+                </Button>
+                {selectedSubject ? (
+                  <Button
+                    onClick={() =>
+                      okButtonClicked(customQuestion, selectedSubject)
+                    }
+                    data-cy="modal-OK-btn"
+                    variant="contained"
+                    color="primary"
+                    component="span"
+                    disabled={
+                      !selectedSubject || !selectedCategory || !customQuestion
```

Comment:
you can remove !selectedSubject from here all together since it's already checked